app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb

```diff
@@ -33,6 +33,12 @@
               description: "As a design and technology teacher, you'll help pupils to turn their ideas into reality.",
               path: "/life-as-a-teacher/explore-subjects/design-and-technology",
             )) %>
+            <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Geography",
+              description: "Find out how to become a geography teacher, including what you'll teach and what funding is available to help you train.",
```

Comment:
This is the same description as the chem and bio cards on the `/life-as-a-teacher/explore-subjects`

I think it is fine for the meta description but could/should we be a bit more engaging on the Explore subjects page for chem/bio/geog?

![image](https://github.com/user-attachments/assets/18641200-fa53-4dce-adf7-387d38cc22de)